unattended.sh

```diff
@@ -1,186 +1,802 @@
+	fi
+
+	#then change the SELinux mode
+	set_runtime_selinux_mode $selinux_mode
+
+	set_selinux_config $selinux_mode
+}
+#========= end of function update_after_installation()
+
+#####################################################
+################ MAIN SCRIPT EXECUTION ##############
+
+## Process the provided arguments in line
+case "$1" in
+
+upgrade)
+	operation="upgrade"
+	parse_subcommand_options "$@"
+	;;
+
+install)
+	operation="install"
+	parse_subcommand_options "$@"
+	;;
+
+*)
+	log "WARN" "No provided operation : default value [$operation] will be used"
+	#usage
+	operation="install"
+	parse_subcommand_options "$@"
+	;;
+
+esac
+
+## Display all configured parameters
+log "INFO" "Start to execute operation [$operation] with following configuration parameters:"
+log "INFO" " topology   : \t[$topology]"
+log "INFO" " version    : \t[$version]"
+log "INFO" " repository : \t[$repo]"
+
+log "WARN" "It will start in [$default_timeout_in_sec] seconds. If you don't want to wait, press any key to continue or Ctrl-C to exit"
+pause "" $default_timeout_in_sec
+
+##
+# Analyze system and set the variables
+##
+set_required_prerequisite
+
+##
+# Check if systemd is present
+##
+is_systemd_present
+
+## Start to execute
+case $operation in
+
+install)
+	setup_before_installation
+	case $topology in
+	central)
+		CENTREON_SELINUX_PACKAGES=(centreon-common-selinux centreon-web-selinux centreon-broker-selinux centreon-engine-selinux centreon-gorgoned-selinux centreon-plugins-selinux)
+		install_central
+		CENTREON_DOC_URL="https://docs.centreon.com/$CENTREON_MAJOR_VERSION/en/installation/web-and-post-installation.html"
+		;;
+
+	poller)
+		CENTREON_SELINUX_PACKAGES=(centreon-common-selinux centreon-broker-selinux centreon-engine-selinux centreon-gorgoned-selinux centreon-plugins-selinux)
+		install_poller
+		CENTREON_DOC_URL="https://docs.centreon.com/$CENTREON_MAJOR_VERSION/en/monitoring/monitoring-servers/add-a-poller-to-configuration.html"
+		;;
+	esac
+
+	update_after_installation
+
+	if [ "x$topology" '=' "xcentral" ]; then
+		if [ "x$wizard_autoplay" '=' "xtrue" ]; then
+			play_install_wizard
+		fi
+	fi
+
+	log "INFO" "Centreon [$topology] successfully installed !"
+
+	if [ "x$topology" '=' "xcentral" ] && [ "x$wizard_autoplay" '=' "xtrue" ]; then
+		log "INFO" "Log in to Centreon web interface via the URL: http://$central_ip/centreon"
+	else
+		log "INFO" "Follow the steps described in Centreon documentation: $CENTREON_DOC_URL"
+	fi
+	;;
+
+upgrade)
+	error_and_exit "Upgrade operation is not supported yet" ##FIXME
+	;;
+esac
+
+## Major change - remind it again (in case of log level is ERROR)
+if [ -e $tmp_passwords_file ] && [ "x$topology" '=' "xcentral" ]; then
+	# Move the tmp file to the dest file
+	mv $tmp_passwords_file $passwords_file
+	echo
+	echo "****** IMPORTANT ******"
+	echo "As you will need passwords for users such as MariaDB [root,centreon] and Centreon [admin], random passwords are generated"
```

Comment:
```suggestion
        if [ "$wizard_autoplay" == "true" ]; then
	        echo "As you will need passwords for users such as MariaDB [root,centreon] and Centreon [admin], random passwords are generated"
	else
	        echo "As you will need password for user MariaDB [root], random password are generated"
	fi
```
Only show more random password if wizard is true